src/blueberrypy/config.py

```diff
@@ -115,23 +133,24 @@ def __init__(self, config_dir=None, app_config=None, logging_config=None,
         if app_config:
             self._app_config = dict(app_config)
 
-        # Merge JSON from environment variable
-        self._app_config = self.__class__.merge_dicts(self._app_config, ENV_CONFIG)
-
-        # Convert relative paths to absolute where needed
-        try:
-            for _ in self._app_config['controllers']:
-                section = self._app_config['controllers'][_]
-                for r in section:
-                    if isinstance(section[r], dict):
-                        for __ in ['tools.staticdir.root',
-                                   'tools.staticfile.root']:
-                            pth = section[r].get(__)
-                            if pth is not None and not pth.startswith('/'):
-                                self._app_config['controllers'][_][r][__] = \
-                                    os.path.join(CWD, pth)
-        except:
-            pass
+        if hasattr(self, '_app_config'):
```

Comment:
Nope. this is not useless. There is a [validate() method call](https://github.com/open-craft-guild/blueberrypy/blob/master/src/blueberrypy/config.py#L142), which [handles](https://github.com/open-craft-guild/blueberrypy/blob/master/src/blueberrypy/config.py#L276-L277) the absence of the config.

BTW, there is a [test case]( https://github.com/open-craft-guild/blueberrypy/blob/master/src/blueberrypy/tests/test_config.py#L52-L55) for that ^_^